package.json

```diff
@@ -48,6 +48,47 @@
     "appId": "org.ssbc.ssb-first-aid-kit",
     "directories": {
       "output": "dist/installers"
+    },
+    "win": {
+      "target": [
+        {
+          "target": "nsis",
+          "arch": [
+            "x64",
+            "ia32"
+          ]
+        }
+      ]
+    },
+    "mac": {
+      "category": "public.app-category.social-networking"
+    },
+    "dmg": {
+      "iconSize": 160,
+      "contents": [
+        {
+          "x": 180, 
```

Comment:
What's this icon stuff? Do we need/want it (since this app doesn't have an icon)?

Reply:
Macs DMGs are virtual disk images that can hold their data in nice organized format. You can add backgrounds, position the icons and size the window that holds them so that when the user mounts the DMG, they see a really nice UI. Those values are positioning the app inside the DMG.

As an example of how it can look, this is a DMG from my job. We have a background, a symlink to `/Applications` and our app, all with a neat layout:

<img width="658" alt="LiveCode Business 9 6 0 (dp 2) 2020-01-15 09-31-21" src="https://user-images.githubusercontent.com/23247/72422218-efaec400-3779-11ea-95bf-44b2fc4649aa.png">

Also, I just noticed that the DMG for Patchwork is built with some nice layout as well:

<img width="652" alt="Patchwork 3 17 2 2020-01-15 09-33-03" src="https://user-images.githubusercontent.com/23247/72422312-1cfb7200-377a-11ea-97c5-7e9b146409dc.png">

Reply:
huh TIL